apps/dolly-frontend/src/main/js/src/components/fagsystem/pdlf/visning/partials/PdlNasjonalitet.js

```diff
@@ -26,7 +26,7 @@ const Statsborgerskap = ({ statsborgerskap }) =>
 	) : null
 
 export const PdlNasjonalitet = ({ data, visTittel = true }) => {
-	const { statsborgerskap, innflyttingTilNorge, utflyttingFraNorge } = data
+	const { statsborgerskap, innflytting: innflyttingTilNorge, utflytting: utflyttingFraNorge } = data
```

Comment:
Litt usikker på hva `innflytting:` og `utflytting:` egentlig gjør, men de sørger ihvertfall for at innflyttingTilNorge og utflyttingFraNorge blir undefined 😉  Så kan de fjernes igjen?